src/main/java/seedu/address/model/ReadOnlyBidderAddressBook.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.person.Bidder;
+
+/**
+ * Unmodifiable view of an address book
+ */
+public interface ReadOnlyBidderAddressBook {
+
+    /**
+     * Returns an unmodifiable view of the persons list.
+     * This list will not contain any duplicate persons.
+     */
+    ObservableList<Bidder> getPersonList();
```

Comment:
Would `getBidderList()` be a better name? So that when you call this method, you know that you are getting a bidder list and likewise for seller list. What do you think?